pygmt/src/text.py

```diff
@@ -223,7 +230,9 @@ def text_(
 
     # Append text at last column. Text must be passed in as str type.
     if kind == "vectors":
-        extra_arrays.append(np.atleast_1d(text).astype(str))
+        extra_arrays.append(
+            list(map(non_ascii_to_octal, np.atleast_1d(text).astype(str)))
```

Comment:
This would be a list of numpy arrays, instead of being just a numpy array? Is there a way to keep it as a numpy array?

Reply:
Changed to `np.vectorize` in 85a0926e457fed9a484dc55845fb87377e258c57.